library/src/main/java/com/xfinity/blueprint/view/ScreenView.kt

```diff
@@ -35,4 +35,5 @@ interface ScreenView {
     fun positionOfFirst(viewType: Int) : Int
     fun updateComponents(newComponents: List<Component>)
     fun updateComponents(newComponents: List<Component>, diffGenerator: (List<Component>) -> DiffUtil.Callback)
+    fun updateCacheSize(size: Int)
```

Comment:
can we make this more specific?  "Cache" by itself isn't obvious for a screen view.

also, we aren't updating anything, they way we are with the diffable component list

suggest "setComponentViewCacheSize"